MarketExecute.php

```diff
@@ -178,6 +178,32 @@ function DefaultErrorHandler($errno, $errstr) {}
                     "ErrorMessage" => "GetAllProducts: unsupported operation"));
             break;
 
+        case "GetNextProducts":
+            #TODO: initialize $StartId with the largest integer possible for database to simplify
+            #TODO:          ProductsExplorer::GetNextProducts($StartId, $ListLength)
+            $StartId = -1;
+            $ListLength = 20;
+            $Array = array();
+
+            if(isset($_POST["StartId"])){
```

Comment:
These 2 parameters (offset and pageSize) are required for this action. Right now, if user doesn't provide them, the program will fail silently.

You might find helper function "CheckPostParam" useful for this purpose.

Reply:
Right now the offset is missing or is invalid, it will return the latest products, and the page size also has a default value. Is there a reason why we require these parameters?